### DISCUSSION TITLE
Foreign language character set problem on spectrograms

### DISCUSSION THREAD

**Discussion by Clumpton** [mcguirepr89/BirdNET-Pi#838] [General]
I have just noticed that accented characters do not show up correctly on the spectrograms, replaced by "xx".
For example:
![Capture](https://user-images.githubusercontent.com/59048696/230899148-dfb0cc7b-2786-4411-bb14-9afe6ce42570.JPG)
Is there an easy fix?

**Comment by ehpersonal38** [comment]
I've just pushed an update to address this. It converts all non-ASCII bird names to ASCII ones for the title of the spectrogram. 
So Mésange charbonnière => Mesange charbonniere.

Take care!

**Reply by Clumpton** [reply]
That'll make it prettier, thanks.
